tests/robottelo/test_hammer.py

```diff
@@ -458,7 +458,7 @@ def test_parse(self):
                 '    Last Checkin: 2019-12-13 00:00:00 UTC',
                 '    Release Version:',
                 '    Autoheal: true',
-                '    Registered To: tier3',
```

Comment:
This does not seem to be a marker. Is this change intentional?

Reply:
it reads to me just like some "random" value, so I replaced it with another.